lib/index.js

```diff
@@ -80,19 +83,32 @@ exports.start = (options) => {
         }
         case 'new': {
 
-            if (!args._[3]) {
+            if (!extraArgs[3]) {
                 throw new DisplayError(colors.red('You must specify a directory in which to start your project.'));
             }
 
             const cwd = options.cwd;
-            const dir = Path.resolve(cwd, args._[3]);
+            const dir = Path.resolve(cwd, extraArgs[3]);
 
             return Commands.new(cwd, dir, ctx)
                 .then(() => {
 
                     output(colors.green(`New pal project created in ${Path.relative(cwd, dir)}`));
                 });
         }
+        case 'run': {
+
+            const cwd = options.cwd;
+            const list = args.list;
+            const cmd = extraArgs[3];
```

Comment:
Do we want to allow any flags with these commands?

Reply:
Oh the flags would come _after_ the `run plugin:cmdName` gotcha

Reply:
Yup!